lib/hexpm_web/controllers/dashboard/audit_log_controller.ex

```diff
@@ -0,0 +1,24 @@
+defmodule HexpmWeb.Dashboard.AuditLogController do
+  use HexpmWeb, :controller
+
+  plug :requires_login
+
+  @per_page 10
```

Comment:
These queries are cheap so let's make this a large number, like `@per_page 100`.

Reply:
updated: https://github.com/hexpm/hexpm/pull/886/commits/e89dfd49bbe15cd34228bf56d6eb6b685d3dd8ed